### ISSUE TITLE
Apalachia cadillions drop empowered meat

### ISSUE THREAD

**Issue by Flameminer221** [DivineRPG/DivineRPG#281]
All meat-dropping Eden mobs (including the Eden cadillion) drop raw Empowered Meat.
All meat-dropping Wildwood mobs (including the Wildwood cadillion) drop regular Magic Meat.
Most meat-dropping Apalachia mobs drop Enriched Magic Meat. However, for some reason, the Apalachia cadillions drop empowered meat, which is inconsistent (especially since they are harder to kill than wildwood cadillions, which currently drop a higher tier of meat than apalachia cadillions do).

In short, are apalachia cadillions supposed to drop empowered meat?
